messages/commonFlags.md

```diff
@@ -50,4 +50,4 @@ Show concise output of the command result.
 
 # errors.NoDefaultDoceEnv
 
-You must specify the DevOps Center org username by indicating the --devops-center-username flag on the command line or by setting the target-devops-center configuration variable.
+Before you run any DevOps Center CLI commands, you must first authorize the org in which DevOps Center is installed. Be sure that the DevOps Center org is authorized, and that you indicated the DevOps Center org username when running the command (--devops-center-username) or set the configuration variable for target-devops-center (sf config set), and try again.
```

Comment:
```suggestion
Before you run any DevOps Center CLI command, you must first use one of the "org login" commands to authorize the org in which DevOps Center is installed. Then, when you run a DevOps Center command, be sure that you specify the DevOps Center org username with the "--devops-center-username" flag. Alternatively, you can set the "target-devops-center" configuration variable to the username with the "config set" command.
```

Reply:
@ekapner : I realize I added more words to an already wordy message.  But there's a lot of information packed in there!  So I tried to parse it out to make it more clear.  Take a look and accept if you agree, reject if you don't.  Your original message is correct, but it felt a little unclear to me.

Reply:
@jshackell-sfdc - I like your updates but made a minor tweak. Thanks.